### PR TITLE
0.12: Wireframe Improvements

### PR DESCRIPTION
**Pull request by IceSentry**

### CODE REVIEW

File: content/news/2023-10-21-bevy-0.12/index.md [hunk 1]
```diff
@@ -15,9 +15,13 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
...
 
-## Feature Name
+## Wireframe Improvements
 
-<div class="release-feature-authors">authors: @author</div>
+<div class="release-feature-authors">authors: @IceSentry</div>
+
+The wireframes now use bevy's Material abstraction. This means it will automatically use the new batching and instancing features while being easier to maintain. This change also made it easier to add support for colored wireframe. You can configure the color globally or per mesh using the `WireframeColor` component. It's also now possible to disable wireframe rendering by using the `NeverRenderWireframe` component.
+
+![wireframe](wireframe.png)
```
Comment (cart) [inline content/news/2023-10-21-bevy-0.12/index.md:24]: I think this example would visually benefit from using slightly less max-RGB coloring for the base colors (and maybe slightly brighter lighting). Ex: something closer to the `3d_scene` palette.

Comment (IceSentry) [reply content/news/2023-10-21-bevy-0.12/index.md:24]: Yeah, good idea. I just used the default wireframe example. I'll update it with nicer colors

Comment (IceSentry) [reply content/news/2023-10-21-bevy-0.12/index.md:24]: How about this?

I used uvspheres and more divisions on the plane to show more wireframe on screen

image

Comment (IceSentry) [reply content/news/2023-10-21-bevy-0.12/index.md:24]: Maybe with a slightly less dense sphere mesh

image

Comment (IceSentry) [reply content/news/2023-10-21-bevy-0.12/index.md:24]: I've pushed that last one. Let me know if you would like any other changes

**Review by alice-i-cecile — APPROVED** [review]

File: content/news/2023-10-21-bevy-0.12/index.md (hunk 1 above)
Comment (cart) [reply content/news/2023-10-21-bevy-0.12/index.md:24]: Sorry for the late reply. Love the changes!

**Review by cart — APPROVED** [review]

**Pull request merged**